### PR TITLE
Top 5 pages - refactor

### PR DESCRIPTION
**Pull request by ankit1khare**
1. server intro
2. http api
3. experiment creation
revamp and corrections

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://66fe116ecdb31ff22c2c0c17--cozy-fox-0defec.netlify.app

**Comment by ankit1khare** [comment]
@tore-statsig : spell checker trouble again

File: docs/http-api.md [hunk 1]
```diff
@@ -2,43 +2,42 @@
...
+---
 
-An account will let you use the Statsig Console, where you can manage all of
-your Feature Gates and Dynamic Configs. Note that you will be able to invite
-others to your Statsig Projects, so they can also interact with your gates and
-configs.
+### Step 2 - Get an API key from the [Statsig Console](https://console.statsig.com/)
 
-<a name="step2"></a>
+An API key is required in every API request. There are two types of API keys:
```
Comment (tore-statsig) [inline docs/http-api.md:21]: two types you can use with the httpapi

File: docs/http-api.md [hunk 2]
```diff
@@ -2,43 +2,42 @@
...
 1. [Create a free account on statsig.com](#step1)
-2. Get an API key from the admin console
-3. Issue API requests
+2. [Get an API key from the admin console](#step2)
+3. [Issue API requests](#step3)
 
-<a name="step1"></a>
+---
+
+### Step 1 - Create a free account on the [Statsig sign-up page](https://console.statsig.com/sign_up)
```
Comment (tore-statsig) [inline docs/http-api.md:13]: we should add a callout that you should avoid the httpapi and use a relevant SDK where possible for a much more performant integration

File: sidebars.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ const sidebars: SidebarsConfig = {
         {
           type: "doc",
           id: "getting-started",
-          label: "Overview",
+          label: "Docs Overview",
```
Comment (tore-statsig) [inline sidebars.ts:17]: I like Overview more actually

**Review by tore-statsig — APPROVED** [review]

File: docs/experiments-plus/create-new.md [hunk 4]
```diff
@@ -4,97 +4,104 @@ sidebar_label: Create
...
+- **Primary Metrics** are those you expect to be directly impacted by the experiment.
+- **Secondary Metrics** are important to monitor to ensure there are no unintended side effects, but they aren’t the primary focus of your experiment.
 
-Configuring the Scorecard is optional, but is especially helpful to ensure other members of your team viewing your experiment have context on the hypothesis being tested and how success is being measured. Additionally, all metrics added to the Scorecard are pre-computed daily, as well as eligible for more advanced statistical treatments like [CUPED](/stats-engine/methodologies/cuped) and [Sequential Testing](/experiments-plus/sequential-testing#what-is-sequential-testing).
+Configuring the Scorecard is optional but recommended. It provides your team with clear context on what is being tested and how success is measured. Metrics added to the Scorecard are computed daily and eligible for advanced treatments like [CUPED](/stats-engine/methodologies/cuped) and [Sequential Testing](/experiments-plus/sequential-testing#what-is-sequential-testing).
```
[the remark this answers is not in the document]
Comment (ankit1khare) [reply docs/experiments-plus/create-new.md:28]: added this:

Configuring the Scorecard is a required step when creating an experiment. It provides your team with clear context on what is being tested and how success is measured. You must enter your hypothesis and select at least one primary metric. Metrics added to the Scorecard are computed daily and eligible for advanced treatments like [CUPED](/stats-engine/methodologies/cuped) and [Sequential Testing](/experiments-plus/sequential-testing#what-is-sequential-testing).

**Pull request merged**